cmd/app.go

```diff
@@ -352,6 +348,22 @@ func appUpgrade(ctx *cli.Context) error {
 	return c.ProjectClient.App.ActionUpgrade(app, au)
 }
 
+func getExternalIDInVersion(externalID string, version string) (string, error) {
```

Comment:
Nit: This function name should change to something like `updateExternalIDVersion`